bridge/client/app/_components/ktb-create-secret-form/ktb-create-secret-form.component.scss

```diff
@@ -0,0 +1,11 @@
+
+/* barista fixes */
+::ng-deep {
+  .dt-form-field-disabled .dt-form-field-prefix {
+    background-color: #f2f2f2;
```

Comment:
Wondering if this is a color that should be fixed via a variable/in the theme?

Reply:
In Barista it's not a variable either, that's why I chose to assign it that way. Else I would have to look for every usage of the disabled field (input, select, etc.) and overwrite it

Reply:
Ok, the color is not set via `var(...)` but there is an existing value for the color (`$gray-130`). Just adapted it